src/style.css

```diff
@@ -1,3 +1,54 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-left: 50px;
+  padding-top: 10px;
+  margin-bottom: 10px;
+}
+
+.nav {
+  display: flex;
+  gap: 20px;
+  font-family: Roboto, sans-serif;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead of applying styles to all `a` elements, consider using a class to style these elements, which would make your styling more modular and reusable.